PennMobile/Login/PathAtPennNetworkManager.swift

```diff
@@ -45,6 +51,49 @@ extension PathAtPennNetworkManager {
 
         return token
     }
+
+    func getToken() async throws -> String {
```

Comment:
We should combine this with the previous method (I don't think this will even compile.) Maybe something like:

1. Try getting the token the old way
2. If that fails, try logging in with the saved username/password
3. If *that* fails, then we throw an error